server/src/main/java/org/elasticsearch/tasks/Traceable.java

```diff
@@ -6,23 +6,14 @@
  * Side Public License, v 1.
  */
 
-package org.elasticsearch.plugins;
+package org.elasticsearch.tasks;
```

Comment:
I wonder if we want to make a dedicated package for tracing (as cluster state and auth tracing is not directly related to tasks)?

Reply:
++ moved to `org.elasticsearch.tracing` in f73b052ff25963590c329f719d778333f2eee503.